docs/references/users/delete.md

```diff
@@ -1 +1 @@
-Delete a user by its unique ID.
\ No newline at end of file
+Delete a user by its unique ID, releasing it's ID. Since ID is released and can be reused, all user-related resources like documents or storage files should be deleted before user deletion. If you want to keep ID reserved, use 'updateStatus' instead.
```

Comment:
```suggestion
Delete a user by its unique ID, thereby releasing it's ID. Since ID is released and can be reused, all user-related resources like documents or storage files should be deleted before user deletion. If you want to keep ID reserved, use the [updateStatus]() endpoint instead.
```

Todo: Add link to the update Status endpoint